packages/dita-example-sx-modules-xsd-topic-mod/src/configureSxModule.ts

```diff
@@ -320,6 +322,36 @@ export default function configureSxModule(sxModule: SxModule): void {
 		}
 	);
 
+	// Needed for bookmap implementation
+	configureProperties(
+		sxModule,
+		xq`self::*[fonto:dita-class(., "topic/topic")][not(parent::*[fonto:dita-class(., "topic/topic")])]`,
+		{
+			titleQuery: xq`
+					let $refNodeName := fonto:hierarchy-source-node(fonto:current-hierarchy-node-id())/name(),
+					$title := ./*[fonto:dita-class(., "topic/title")]//text()[not(ancestor::*[name() = ("sort-at", "draft-comment", "foreign", "unknown", "required-cleanup", "image")])]/string() => string-join()
```

Comment:
You can replace this madness with `$title := fonto:curated-text-in-node(.)` btw. This pattern is old and useless

Reply:
Modified it to use `curated-text-in-node()` while still using `./*[fonto:dita-class(., "topic/title")]`.